src/Components/Home/BackOffice/Categories.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { Link } from "react-router-dom"
+import "./stylesTable.css"
+
+export default function Categories() {
+
+    const [categories, setCategories] = useState([
+        { id: 1, name: 'Categoria-1', createdAt: '2022-01-03' },
+        { id: 2, name: 'Categoria-2', createdAt: '2022-01-20' },
+        { id: 3, name: 'Categoria-3', createdAt: '2022-01-31' }
+    ]);
+
+    const handleClickUpdate = (id) => {}
+
+    const handleClickDelete = (id) => {}
+
+    return (
+        <section className="sectionTable">
+            <table className="table">
+                <thead>
+                    <tr>
+                        <td>
+                            <Link to="/backoffice/categories/create" className="btnAddTable">Crear</Link>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th>Nombre</th>
+                        <th>Creado</th>
+                        <th>Acciones</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {categories.map(category => {
+                        return (
+                            <tr key = {category.id}>
+                                <td>{category.name}</td>
+                                <td>{category.createdAt}</td>
+                                <td>
+                                    <button className="btnUpdateTable" type = "submit" onClick = {() => handleClickUpdate(category.id)}>Editar</button>
+                                    <button className="btnDeleteTable" type = "submit" onClick = {() => handleClickDelete(category.id)}>Eliminar</button>
```

Comment:
Ojo los botones, le marqué el mismo tema a @EstebanAbella. El type submit de los botones es solamente para envío de formularios, si lo que van a hacer es ejecutar una función fuera del contexto de un form, como en este caso, usar type button.

```suggestion
                                    <button className="btnUpdateTable" type="button" onClick={() => handleClickUpdate(category.id)}>Editar</button>
                                    <button className="btnDeleteTable" type="button" onClick={() => handleClickDelete(category.id)}>Eliminar</button>
```